test/unittests/t_sqlitemem.cc

```diff
@@ -0,0 +1,254 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include <gtest/gtest.h>
+
+// #include <malloc.h> for malloc_usable_size
+
+#include <cstdlib>
+#include <cstring>
+#include <map>
+#include <vector>
+
+#include "../../cvmfs/murmur.h"
+#include "../../cvmfs/prng.h"
+#include "../../cvmfs/sqlitemem.h"
+#include "../../cvmfs/util.h"
+
+using namespace std;  // NOLINT
+
+namespace sqlite {
+
+class T_Sqlitemem : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    EXPECT_FALSE(MemoryManager::HasInstance());
+    mem_mgr_ = MemoryManager::GetInstance();
+    EXPECT_TRUE(MemoryManager::HasInstance());
```

Comment:
potentially `ASSERT_TRUE` makes more sense here... if the previous call fails there is no point in running the test...